ee/vellum_cli/tests/test_pull.py

```diff
@@ -132,6 +133,7 @@ def test_pull__sandbox_id_with_no_config(vellum_client):
                     "deployments": [],
                     "container_image_tag": None,
                     "container_image_name": None,
+                    "workspace": "default",
```

Comment:
In a future PR, I'm going to look into how to not have default values written into the state lock file